src/manifest/manifest.ts

```diff
@@ -139,22 +139,31 @@ export default class Manifest extends EventEmitter<IManifestEvents> {
   // begins.
   public availabilityStartTime? : number;
 
-  // Informations about the first seekable position.
+  /**
+   * Informations about the first seekable position.
+   * @type {Object|undefined}
+   */
```

Comment:
Why?

Reply:
(I voluntarly removed those lines as they cluttered the file too much for no real value)